src/test/java/pages_sample/ShopPage.java

```diff
@@ -1,27 +1,27 @@
 package pages_sample;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.How;
 
-import static org.junit.Assert.*;
-
 public class ShopPage {
 
-    @FindBy(how = How.ID, using = "name") // By.id("name")
-    private WebElement nameInput; // WebElement nameInput = driver.findElement(By.id("name"));
-    @FindBy(how = How.NAME, using = "age") // By.name("age")
-    private WebElement ageInput;
-    @FindBy(how = How.ID, using = "submit")
-    private WebElement submitButton;
-    @FindBy(how = How.CLASS_NAME, using = "error") // By.className("error)
-    private WebElement errorText;
-    @FindBy(how = How.XPATH, using = "//strong[contains(text(),'0123-456-789')]")
+    @FindBy(how = How.ID, using = "SubmitLogin")
+    private WebElement submitLoginButton;
+    @FindBy(how = How.CSS, using = ".shop-phone strong")
     private WebElement topPhone;
-    @FindBy(how = How.XPATH, using = "//span[contains(text(),'(347) 466-7432')]")
+    @FindBy(how = How.CSS, using = "#block_contact_infos > div > ul > li:nth-child(2) > span")
     private WebElement bottomPhone;
+    @FindBy(how = How.ID, using = "email") // By.id("name")
+    private WebElement inputEmail; // WebElement nameInput = driver.findElement(By.id("name"));
```

Comment:
If I am not mistaken, this elements are on different page. Why are they in this class?